internal/app/server/open_saves.go

```diff
@@ -103,6 +103,7 @@ func (s *openSavesServer) CreateStore(ctx context.Context, req *pb.CreateStoreRe
 		return nil, status.Convert(err).Err()
 	}
 	log.Debugf("Created store: %+v", store)
+	s.storeCache(ctx, &store)
```

Comment:
Can you please catch the error and log a warning here?

Reply:
the warning will be thrown in the dowstream function check line #1037